ISS/views/forum.py

```diff
@@ -90,6 +91,9 @@ def thread(request, thread_id):
     reply_form = _get_new_post_form(request)(author=request.user,
                                              initial={ 'thread': thread })
 
+    if thread.forum.member_view_only and not request.user.is_authenticated:
+        return render(request, 'members_only.html')
```

Comment:
I like the specific page telling them what happened as opposed to the generic 403 page, but we should still answer with a 403 status code